src/ucp/api/ucp.h

```diff
@@ -357,6 +357,11 @@ typedef struct ucp_params {
      * This field defaults to 0 if not specified.
      */
     uint64_t                           tag_sender_mask;
+    /**
+     * Thread access mode to the context.
+     * This field defaults to UCS_THREAD_MODE_SINGLE if not specified.
+     */
+    ucs_thread_mode_t                  thread_mode;
```

Comment:
@yosefe - I don't like us referencing ucs* contracts in UCP.H file. Shell we move ucs_thread_mode_t to UCP ?